pkgs/development/python-modules/openai-triton/default.nix

```diff
@@ -5,7 +5,9 @@
 , fetchFromGitHub
 , addOpenGLRunpath
 , cmake
+, config
 , cudaPackages ? { }
+, cudaSupport ? config.cudaSupport or false
```

Comment:
Can you define this as a proper option in the file that describes the config attr? Then we also have a default false

Reply:
Oh, so `config` already supports options?!!

Reply:
Looking at `pkgs/top-level/config.nix` for the first time now. This is great, I really needed this, our `cudaCapabilities` and `cudaSupport` are already a mess, and we still need more controls (virtual vs real architectures, default mkl support)

Reply:
We'll have a separate PR for `config.nix` integration

Reply:
> Oh, so `config` already supports options?!!

It does since ~4 years but people only really started using it a year ago. Glad that that helps you.